bin/pycbc_condition_strain

```diff
@@ -56,6 +56,9 @@ parser.add_argument('--low-frequency-cutoff', type=float,
                     help='Provide a low-frequency-cutoff for fake strain. '
                          'This is only needed if fake-strain or '
                          'fake-strain-from-file is used')
+parser.add_argument('--frame-duration', type=int, default=0,
```

Comment:
Small pet peeve: remove `default=0` here, and replace the check on line 89 with `if args.frame_duration is None:`